### ISSUE TITLE
Another issue with the Lilypad Mixin...?

### ISSUE THREAD

**Issue by IdrisQe** [Hephaestus-Dev/TinyTweaks#63]
I've had this error several times now, and I can't figure out WHY. It crashes on launch attempt but only seems to happen after I've fiddled with my mod setup in some way... This time it was after I removed Mouse Wheelie to test stuff, but Mouse Wheelie doesn't touch Lilypads at all, heck it's completely clientside. I don't understand why this is occuring only after I remove that though... I guess it may have to do with the order mods are getting loaded in or something? I have no idea.

```
[15:55:24] [main/FATAL]: Mixin apply failed tweaks.mixins.json:entity.betterlilypads.FallingParticles -> net.minecraft.class_1309: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException Variable modifier target for net/minecraft/class_1309::changeLandedState was removed by another injector [INJECT Applicator Phase -> tweaks.mixins.json:entity.betterlilypads.FallingParticles -> Apply Injections ->  -> Inject -> tweaks.mixins.json:entity.betterlilypads.FallingParticles->@ModifyVariable::changeLandedState(Lnet/minecraft/class_2680;DZLnet/minecraft/class_2680;Lnet/minecraft/class_2338;)Lnet/minecraft/class_2680;]
org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: Variable modifier target for net/minecraft/class_1309::changeLandedState was removed by another injector [INJECT Applicator Phase -> tweaks.mixins.json:entity.betterlilypads.FallingParticles -> Apply Injections ->  -> Inject -> tweaks.mixins.json:entity.betterlilypads.FallingParticles->@ModifyVariable::changeLandedState(Lnet/minecraft/class_2680;DZLnet/minecraft/class_2680;Lnet/minecraft/class_2338;)Lnet/minecraft/class_2680;]
	at org.spongepowered.asm.mixin.injection.modify.ModifyVariableInjector.inject(ModifyVariableInjector.java:143) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.injection.code.Injector.inject(Injector.java:264) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.inject(InjectionInfo.java:385) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.applyInjections(MixinTargetContext.java:1284) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyInjections(MixinApplicatorStandard.java:1042) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:395) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:320) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:345) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:569) [sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:351) [sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:208) [sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:178) [sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.FabricMixinTransformerProxy.transformClassBytes(FabricMixinTransformerProxy.java:23) [fabric-loader-0.11.2.jar:0.9.2+mixin.0.8.2]
	at net.fabricmc.loader.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:157) [fabric-loader-0.11.2.jar:?]
	at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:150) [fabric-loader-0.11.2.jar:?]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.8.0_281]
	at java.lang.ClassLoader.defineClass1(Native Method) [?:1.8.0_281]
	at java.lang.ClassLoader.defineClass(Unknown Source) [?:1.8.0_281]
	at java.security.SecureClassLoader.defineClass(Unknown Source) [?:1.8.0_281]
	at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:163) [fabric-loader-0.11.2.jar:?]
	at java.lang.ClassLoader.loadClass(Unknown Source) [?:1.8.0_281]
	at carpetextra.CarpetExtraServer.<clinit>(CarpetExtraServer.java:26) [carpet-extra-1.16.5-1.4.27.jar:?]
	at net.minecraft.class_128.handler$cdf000$gameStarted(class_128.java:2016) [intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_128.method_24305(class_128.java) [intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:135) [intermediary-minecraft-1.16.5-client.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_281]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_281]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_281]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_281]
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226) [fabric-loader-0.11.2.jar:?]
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:146) [fabric-loader-0.11.2.jar:?]
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28) [fabric-loader-0.11.2.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_281]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_281]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_281]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_281]
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196) [NewLaunch.jar:?]
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.listen(EntryPoint.java:143) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.main(EntryPoint.java:34) [NewLaunch.jar:?]
Failed to start Minecraft:
java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)
Caused by: java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:228)
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:146)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28)
	... 8 more
Caused by: java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226)
	... 10 more
Caused by: org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:208)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:178)
	at org.spongepowered.asm.mixin.transformer.FabricMixinTransformerProxy.transformClassBytes(FabricMixinTransformerProxy.java:23)
	at net.fabricmc.loader.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:157)
	at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:150)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClass(Unknown Source)
	at java.security.SecureClassLoader.defineClass(Unknown Source)
	at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:163)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	at carpetextra.CarpetExtraServer.<clinit>(CarpetExtraServer.java:26)
	at net.minecraft.class_128.handler$cdf000$gameStarted(class_128.java:2016)
	at net.minecraft.class_128.method_24305(class_128.java)
	at net.minecraft.client.main.Main.main(Main.java:135)
	... 15 more
Caused by: org.spongepowered.asm.mixin.throwables.MixinApplyError: Mixin [tweaks.mixins.json:entity.betterlilypads.FallingParticles] from phase [DEFAULT] in config [tweaks.mixins.json] FAILED during APPLY
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinError(MixinProcessor.java:642)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinApplyError(MixinProcessor.java:594)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:356)
	... 30 more
Caused by: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: Variable modifier target for net/minecraft/class_1309::changeLandedState was removed by another injector [INJECT Applicator Phase -> tweaks.mixins.json:entity.betterlilypads.FallingParticles -> Apply Injections ->  -> Inject -> tweaks.mixins.json:entity.betterlilypads.FallingParticles->@ModifyVariable::changeLandedState(Lnet/minecraft/class_2680;DZLnet/minecraft/class_2680;Lnet/minecraft/class_2338;)Lnet/minecraft/class_2680;]
	at org.spongepowered.asm.mixin.injection.modify.ModifyVariableInjector.inject(ModifyVariableInjector.java:143)
	at org.spongepowered.asm.mixin.injection.code.Injector.inject(Injector.java:264)
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.inject(InjectionInfo.java:385)
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.applyInjections(MixinTargetContext.java:1284)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyInjections(MixinApplicatorStandard.java:1042)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:395)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:320)
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:345)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:569)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:351)
	... 30 more
```

**Comment by Haven-King** [comment]
Can you provide your mods list please?

**Comment by IdrisQe** [comment]
Oh boy... I was hoping that wouldn't be necessary.
Better idea: I'll take some time to try to squish it down into roughly as small a pack as I can where the error still occurs, because I highly doubt you want to sift through the 124 mods I'm using (granted like half of them are tiny, single-object tweaks or library APIs)

I'll get back to you with a more reasonable list shortly!

**Comment by IdrisQe** [comment]
Got it down to 38 mods (well... mods and APIs. Bewitchment alone needs 7 of the things). No matter which one I try to remove past this point, the error seems to stop occuring, and I don't know why... This is becoming reminiscent of the issues I had with BackSlot last night... This modpack is totally cursed.

```
Allure 1.2.8
Architectury 1.8.128
Bedrockify 0.2.5
BetterEnd 0.9.1-pre
Bewitchment 1.16.5-3
Carpet 1.4.28+v210224
Carpet Extra 1.4.27
Charm 2.3.2
Cleancut 2.1
Cloth Config 4.11.14
Dawn API 1.7
Fabric API 0.31.0
Fabrication 1.3.3
Fabric Language Kotlin 1.4.30+build.2
FerriteCore 2.0.3
Full Slabs 1.2.0
GeckoLib 3.0.22
Impersonate 2.1.0
Krypton 0.1.2
Lakeside 1.0.3
Lithium 0.6.4
Mini Extras 1.2.1
ModMenu 1.16.8
NotEnoughCrashes 3.1.7
Patchouli 1.16.4-50
Pehkui 1.8.0+21w08b
Phosphor 0.7.1+build.11
RandomPatches 2.4.5
River Redux 0.1.6
SmoothBoot 1.4.0
Somnus API 0.0.11
Terrestria 2.2.0
TinyTweaks 1.2.3
Traverse 3.3.2
Trinkets 2.6.7
VanillaTweaks 1.0.6
```

**Comment by IdrisQe** [comment]
Also here's my latest.log from the latest launch crash with just those 38 mods.
```
[17:47:22] [main/INFO]: Loading for game Minecraft 1.16.5
[17:47:22] [ForkJoinPool-1-worker-11/WARN]: The mod "aquarius" contains invalid entries in its mod json:
- Unsupported root entry "licence" at line 9 column 12
[17:47:22] [ForkJoinPool-1-worker-2/WARN]: The mod "autoconfig1u" contains invalid entries in its mod json:
- Unsupported root entry "$schema" at line 2 column 14
[17:47:22] [ForkJoinPool-1-worker-2/WARN]: The mod "autoconfig1u" contains invalid entries in its mod json:
- Unsupported root entry "$schema" at line 2 column 14
[17:47:23] [ForkJoinPool-1-worker-2/WARN]: The mod "cardinal-components-base" contains invalid entries in its mod json:
- Unsupported root entry "side" at line 8 column 11
[17:47:23] [ForkJoinPool-1-worker-2/WARN]: The mod "cardinal-components-entity" contains invalid entries in its mod json:
- Unsupported root entry "side" at line 8 column 11
[17:47:23] [main/INFO]: [FabricLoader] Loading 116 mods: fabric-keybindings-v0@0.2.1+ca58154a7d, notenoughcrashes@3.1.7, trinkets@2.6.7, fabricloader@0.11.2, org_jetbrains_kotlinx_kotlinx-serialization-json-jvm@1.0.1, riverredux@0.1.6, fabric-dimensions-v1@2.0.6+9354966b7d, java@8, cotton-client-commands@1.0.1+1.16-rc1, fabric-game-rule-api-v1@1.0.6+ca58154a7d, fabric-api-base@0.2.1+9354966b7d, fiber@0.23.0-2, bewitchment@1.16.5-3, fabrication@1.3.3, randompatches@2.4.5-fabric, org_jetbrains_kotlin_kotlin-stdlib@1.4.30, com_moandjiezana_toml_toml4j@0.7.2, terraform-dirt-api-v1@1.1.3, fabric-blockrenderlayer-v1@1.1.5+ca58154a7d, terraform-wood-api-v1@1.0.4, org_jetbrains_kotlin_kotlin-stdlib-jdk7@1.4.30, carpet-extra@1.4.27, fabric@0.31.0+1.16, pride@1.0.0+1.16, fabric-networking-v0@0.3.2+ca58154a7d, dawn@1.7, fabric-screen-handler-api-v1@1.1.5+9354966b7d, cleancut@mc1.16-2.1-fabric, modmenu@1.16.8, phosphor@0.7.1+build.11, org_jetbrains_kotlin_kotlin-stdlib-jdk8@1.4.30, fabric-tool-attribute-api-v1@1.2.6+ca58154a7d, terraform-tree-api-v1@1.0.5, fabric-events-interaction-v0@0.4.2+ca58154a7d, fabric-crash-report-info-v1@0.1.3+ca58154a7d, fabric-language-kotlin@1.4.30+build.2, patchouli@1.16.4-50-FABRIC, fabric-key-binding-api-v1@1.0.4+9354966b7d, cloth-config2@4.11.14, org_jetbrains_kotlinx_kotlinx-coroutines-jdk8@1.4.2, fabric-resource-loader-v0@0.4.2+ca58154a7d, org_jetbrains_kotlin_kotlin-reflect@1.4.30, cardinal-components-entity@2.7.10, architectury@1.8.128, fabric-registry-sync-v0@0.7.4+ca58154a7d, fabric-commands-v0@0.2.2+ca58154a7d, cardinal-components-base@2.7.10, fabric-mining-levels-v0@0.1.3+ca58154a7d, fabric-networking-api-v1@1.0.1+ca58154a7d, krypton@0.1.2, charm@2.3.2, fabric-models-v0@0.2.1+ca58154a7d, fabric-item-groups-v0@0.2.3+ca58154a7d, geckolib3@3.0.22, bedrockify@0.2.5, terraform-shapes-api-v1@1.0.4, fabric-renderer-api-v1@0.4.1+ca58154a7d, terrestria@2.2.0, full_slabs@1.2.0, biomemakeover@1.16.4-1.2.11, fabric-structure-api-v1@1.1.4+ca58154a7d, allure@1.2.8, fabric-containers-v0@0.1.11+9354966b7d, lithium@0.6.4, fabric-object-builder-api-v1@1.9.4+9354966b7d, pehkui@1.8.0+21w08b, impersonate@2.1.0, terraform-config-api-v1@1.0.3, tinytweaks@1.2.3, fabric-rendering-data-attachment-v1@0.1.5+ca58154a7d, org_jetbrains_kotlinx_kotlinx-serialization-core-jvm@1.0.1, terraform-surfaces-api-v1@1.0.3, vanillatweaks@1.16.4-1.0.6, fabric-textures-v0@1.0.6+ca58154a7d, com_electronwill_night-config_core@3.6.3, fabric-rendering-fluids-v1@0.1.13+ca58154a7d, reach-entity-attributes@1.0.1, fabric-lifecycle-events-v1@1.2.1+ca58154a7d, traverse@3.3.2, fabric-renderer-registries-v1@2.2.1+ca58154a7d, fabric-loot-tables-v1@1.0.2+ca58154a7d, carpet@1.4.28, fabric-particles-v1@0.2.4+ca58154a7d, recipeconditions@0.4.0, autoconfigtoml@1.0.0-fabric, terraform-biome-builder-api-v1@1.0.4, fabric-networking-blockentity-v0@0.2.8+ca58154a7d, minecraft@1.16.5, jankson@3.0.1+j1.2.0, cloth-basic-math@0.5.1, fabric-command-api-v1@1.1.0+f9b3753b7d, fabric-screen-api-v1@1.0.0+c045166c7d, miniextras@1.2.1, fabric-renderer-indigo@0.4.4+ca58154a7d, org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm@1.4.2, smoothboot@1.16.4-1.4.0, com_electronwill_night-config_toml@3.6.3, fabric-item-api-v1@1.2.1+ca58154a7d, somnus@0.0.11-1.16.5, libgui@3.0.0-beta.1+1.16.2-rc2, fabric-entity-events-v1@1.0.3+ca58154a7d, fabric-rendering-v0@1.1.2+ca58154a7d, fabric-rendering-v1@1.5.1+ca58154a7d, terraform-overworld-biome-extensions-api-v1@1.0.3, autoconfig1u@3.3.1, fabric-content-registries-v0@0.2.1+ca58154a7d, fabric-tag-extensions-v0@1.1.1+ca58154a7d, fabric-biome-api-v1@3.1.1+ca58154a7d, ferritecore@2.0.3, lakeside@1.0.3+1.16.4, fabric-events-lifecycle-v0@0.2.1+ca58154a7d, aquarius@1.0.3, libstructure@1.5, climbable@0.2.0, betterend@0.9.1-pre, fabric-object-builders-v0@0.7.2+ca58154a7d
[17:47:23] [main/WARN]: Mod `cleancut` (mc1.16-2.1-fabric) does not respect SemVer - comparison support is limited.
[17:47:23] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.2 Source=file:/C:/Users/Idris/Documents/My%20Games/Minecraft/MultiMC/libraries/net/fabricmc/sponge-mixin/0.9.2+mixin.0.8.2/sponge-mixin-0.9.2+mixin.0.8.2.jar Service=Knot/Fabric Env=CLIENT
[17:47:23] [main/INFO]: Loaded configuration file for Lithium: 80 options available, 1 override(s) found
[17:47:23] [main/INFO]: [Fabrication] Reloading configs...
[17:47:23] [main/INFO]: [Fabrication] Loading of features.ini done in 8.256 ms with 0 warnings
[17:47:23] [main/WARN]: Overriding InjectionInfo for @Inject with com.unascribed.fabrication.support.injection.FailsoftCallbackInjectionInfo (previously org.spongepowered.asm.mixin.injection.struct.CallbackInjectionInfo)
[17:47:23] [main/WARN]: Overriding InjectionInfo for @ModifyArg with com.unascribed.fabrication.support.injection.FailsoftModifyArgInjectionInfo (previously org.spongepowered.asm.mixin.injection.struct.ModifyArgInjectionInfo)
[17:47:23] [main/WARN]: Overriding InjectionInfo for @ModifyArgs with com.unascribed.fabrication.support.injection.FailsoftModifyArgsInjectionInfo (previously org.spongepowered.asm.mixin.injection.struct.ModifyArgsInjectionInfo)
[17:47:23] [main/WARN]: Overriding InjectionInfo for @Redirect with com.unascribed.fabrication.support.injection.FailsoftRedirectInjectionInfo (previously org.spongepowered.asm.mixin.injection.struct.RedirectInjectionInfo)
[17:47:23] [main/WARN]: Overriding InjectionInfo for @ModifyVariable with com.unascribed.fabrication.support.injection.FailsoftModifyVariableInjectionInfo (previously org.spongepowered.asm.mixin.injection.struct.ModifyVariableInjectionInfo)
[17:47:23] [main/WARN]: Overriding InjectionInfo for @ModifyConstant with com.unascribed.fabrication.support.injection.FailsoftModifyConstantInjectionInfo (previously org.spongepowered.asm.mixin.injection.struct.ModifyConstantInjectionInfo)
[17:47:23] [main/WARN]: Reference map 'pride-refmap.json' for pride.mixins.json could not be read. If this is a development environment you can ignore this message
[17:47:24] [main/WARN]: @Mixin target net/minecraft/class_4019$class_4024 is public in pehkui.mixins.json:compat116plus.FoxEntityMateGoalMixin and should be specified in value
[17:47:24] [main/WARN]: Force-disabling mixin 'world.player_chunk_tick.ThreadedAnvilChunkStorageMixin' as rule 'mixin.world.player_chunk_tick' (added by mods [krypton]) disables it and children
[17:47:24] [main/WARN]: @Mixin target net/minecraft/class_3962$class_3963 is public in lithium.mixins.json:alloc.composter.ComposterMixin$ComposterBlockComposterInventoryMixin and should be specified in value
[17:47:24] [main/WARN]: @Mixin target net/minecraft/class_4970$class_4971$class_3752 is public in ferritecore.blockstatecache.mixin.json:BlockStateCacheAccess and should be specified in value
[17:47:24] [main/WARN]: @Mixin target net/minecraft/class_3898$class_3208 is public in mixins.impersonate.common.json:EntityTrackerAccessor and should be specified in value
[17:47:24] [main/WARN]: @Mixin target net/minecraft/class_4019$class_4024 is public in carpet-extra.mixins.json:FoxMateGoalMixin and should be specified in value
[17:47:24] [main/WARN]: @Mixin target net/minecraft/class_1481$class_1482 is public in carpet-extra.mixins.json:TurtleMateGoalMixin and should be specified in value
[17:47:24] [main/WARN]: @Mixin target net/minecraft/class_3898$class_3208 is public in fabric-networking-api-v1.mixins.json:accessor.EntityTrackerAccessor and should be specified in value
[17:47:25] [main/INFO]: Smooth Boot config initialized
[17:47:26] [main/WARN]: @Inject(@At("INVOKE")) Shift.BY=2 on modid.mixins.json:events.MixinLivingEntity::handler$bfn000$entityEquip exceeds the maximum allowed value: 0. Increase the value of maxShiftBy to suppress this warning.
[17:47:26] [main/INFO]: Instancing error handler class com.unascribed.fabrication.support.MixinErrorHandler
[17:47:26] [main/FATAL]: Mixin apply failed tweaks.mixins.json:entity.betterlilypads.FallingParticles -> net.minecraft.class_1309: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException Variable modifier target for net/minecraft/class_1309::changeLandedState was removed by another injector [INJECT Applicator Phase -> tweaks.mixins.json:entity.betterlilypads.FallingParticles -> Apply Injections ->  -> Inject -> tweaks.mixins.json:entity.betterlilypads.FallingParticles->@ModifyVariable::changeLandedState(Lnet/minecraft/class_2680;DZLnet/minecraft/class_2680;Lnet/minecraft/class_2338;)Lnet/minecraft/class_2680;]
org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: Variable modifier target for net/minecraft/class_1309::changeLandedState was removed by another injector [INJECT Applicator Phase -> tweaks.mixins.json:entity.betterlilypads.FallingParticles -> Apply Injections ->  -> Inject -> tweaks.mixins.json:entity.betterlilypads.FallingParticles->@ModifyVariable::changeLandedState(Lnet/minecraft/class_2680;DZLnet/minecraft/class_2680;Lnet/minecraft/class_2338;)Lnet/minecraft/class_2680;]
	at org.spongepowered.asm.mixin.injection.modify.ModifyVariableInjector.inject(ModifyVariableInjector.java:143) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.injection.code.Injector.inject(Injector.java:264) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.inject(InjectionInfo.java:385) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.applyInjections(MixinTargetContext.java:1284) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyInjections(MixinApplicatorStandard.java:1042) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:395) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:320) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:345) ~[sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:569) [sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:351) [sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:208) [sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:178) [sponge-mixin-0.9.2+mixin.0.8.2.jar:0.9.2+mixin.0.8.2]
	at org.spongepowered.asm.mixin.transformer.FabricMixinTransformerProxy.transformClassBytes(FabricMixinTransformerProxy.java:23) [fabric-loader-0.11.2.jar:0.9.2+mixin.0.8.2]
	at net.fabricmc.loader.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:157) [fabric-loader-0.11.2.jar:?]
	at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:150) [fabric-loader-0.11.2.jar:?]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.8.0_281]
	at java.lang.ClassLoader.defineClass1(Native Method) [?:1.8.0_281]
	at java.lang.ClassLoader.defineClass(Unknown Source) [?:1.8.0_281]
	at java.security.SecureClassLoader.defineClass(Unknown Source) [?:1.8.0_281]
	at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:163) [fabric-loader-0.11.2.jar:?]
	at java.lang.ClassLoader.loadClass(Unknown Source) [?:1.8.0_281]
	at carpetextra.CarpetExtraServer.<clinit>(CarpetExtraServer.java:26) [carpet-extra-1.16.5-1.4.27.jar:?]
	at net.minecraft.class_128.handler$bif000$gameStarted(class_128.java:2016) [intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.class_128.method_24305(class_128.java) [intermediary-minecraft-1.16.5-client.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:135) [intermediary-minecraft-1.16.5-client.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_281]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_281]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_281]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_281]
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226) [fabric-loader-0.11.2.jar:?]
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:146) [fabric-loader-0.11.2.jar:?]
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28) [fabric-loader-0.11.2.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_281]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_281]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_281]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_281]
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196) [NewLaunch.jar:?]
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.listen(EntryPoint.java:143) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.main(EntryPoint.java:34) [NewLaunch.jar:?]

```
